src/Tribe/Utils/Post_Thumbnail.php

```diff
@@ -286,17 +286,22 @@ public function serialize() {
 		$data            = $this->fetch_data();
 		$data['post_id'] = $this->post_id;
 
-		return serialize( $data );
+		return wp_json_encode( $data );
 	}
 
 	/**
 	 * {@inheritDoc}
 	 */
 	public function unserialize( $serialized ) {
-		$data          = unserialize( $serialized );
+		$data          = json_decode( $serialized, true );
+		array_walk( $data, static function ( &$data_entry ) {
+			if ( is_array( $data_entry ) ) {
+				$data_entry = (object)$data_entry;
+			}
+		} );
 		$this->post_id = $data['post_id'];
 		unset( $data['post_id'] );
-		$this->data = $data;
+		$this->data = ! empty( $data ) ? $data : null;
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: TRUE, FALSE and NULL must be uppercase; expected "NULL" but found "null"

<sub>hash: 5d481c8514499ef575d00c276ba90aec</sub>